extension/src/server/interactionService.ts

```diff
@@ -248,6 +248,12 @@ export class InteractionService implements IInteractionService {
     async displayDashboardUrls(dashboardUrls: DashboardUrls) {
         extensionLogOutputChannel.info(`Displaying dashboard URLs: ${JSON.stringify(dashboardUrls)}`);
 
+        this.writeDebugSessionMessage(dashboard + ': ' + dashboardUrls.BaseUrlWithLoginToken, true, '\x1b[32m');
+
+        if (dashboardUrls.CodespacesUrlWithLoginToken) {
+            this.writeDebugSessionMessage(codespaces + ': ' + dashboardUrls.CodespacesUrlWithLoginToken, true, '\x1b[32m');
+        }
```

Comment:
Move '\x1b[32m' to a readonly field somewhere. It's much clearer what it is if it has a name